src/db.py

```diff
@@ -87,7 +91,7 @@ def get_valid_jobs(self, username_filter: Optional[str]) -> List[Job]:
         with self.session() as session:
             query = (
                 select(Job)
-                .where(Job.status != "removed")
+                .where(Job.status != JobStatus.cancelled)
```

Comment:
No need for this where - idea of #414 is to remove all jobs with this status from the database. So there are no more jobs that were removed and need to be filtered out.
```suggestion
```